src/main/kotlin/com/vk/admstorm/playground/KphpPlaygroundWindow.kt

```diff
@@ -128,14 +129,20 @@ require_once 'vendor/autoload.php';
 
             executeOnPooledThread {
                 val link = HastebinService.getInstance(myProject).createHaste(content)
-                MyUtils.copyToClipboard(link)
+                if (link != null) {
```

Comment:
if `link` equals `null`, then show the notification and exit

Reply:
Done